File: templates/Uwp/_composition/MVVMBasic/Page.AddBackCommand._VB/ViewModels/wts.ItemNameDetailViewModel_postaction.vb
'{[{
Imports Param_RootNamespace.Services
'}]}
Namespace ViewModels
    Public Class wts.ItemNameDetailViewModel
        Inherits Observable

        '^^
        '{[{

        Public ReadOnly Property GoBackCommand As ICommand = new RelayCommand(AddressOf OnGoBack)
        '}]}

        Public Sub New()
        End Sub

        '^^
        '{[{
        Private Sub OnGoBack()
            If NavigationService.CanGoBack Then
                NavigationService.GoBack()
            End If
        End Sub
        '}]}
    End Class
End Namespace

File: templates/Uwp/_comp/MVVMBasic/Page.AddBackCommand/ViewModels/wts.ItemNameDetailViewModel_postaction.cs
//{[{
using System.Windows.Input;
using Param_RootNamespace.Helpers;
using Param_RootNamespace.Services;
//}]}

namespace Param_RootNamespace.ViewModels
{
    public class wts.ItemNameDetailViewModel : Observable
    {
//^^
//{[{
        private ICommand _goBackCommand;

        public ICommand GoBackCommand => _goBackCommand ?? (_goBackCommand = new RelayCommand(OnGoBack));
//}]}

        public wts.ItemNameDetailViewModel()
        {
        }

//^^
//{[{
        private void OnGoBack()
        {
            if (NavigationService.CanGoBack)
            {
                NavigationService.GoBack();
            }
        }
//}]}
    }
}